components/atom/input/src/Input.js

```diff
@@ -0,0 +1,86 @@
+/**
+ * This component is used as prop filter for the native input
+ */
+import React from 'react'
+import PropTypes from 'prop-types'
+
+const Input = ({
```

Comment:
This component works as a prop filter as it is not possible to just do `<input {...props} />`.We would pass props that are not recognized as valid input tag props making react to throw warnings and ignore those props (which would be fatal in case of the `reference` prop used in the mask input, for instance). It also avoids painful (IMO) manual prop handling in each input type.

We could add all attributes defined in https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input